src/Features/Core/Portable/Completion/CompletionServiceWithProviders.cs

```diff
@@ -194,7 +194,11 @@ internal protected CompletionProvider GetProvider(CompletionItem item)
             {
                 lock (_gate)
                 {
-                    _nameToProvider.TryGetValue(name, out provider);
+                    if (!_nameToProvider.TryGetValue(name, out provider))
+                    {
+                        CreateRoleProviders(ImmutableHashSet.Create<string>());
```

Comment:
I guess this is normally initialized by the call to GetProviders? Why isn't that getting called?